<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Camp Parameters</string>
     </attribute>
     <widget class="QSpinBox" name="RedBalancespinBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7999</number>
      </property>
     </widget>
     <widget class="QComboBox" name="SceneModecomboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ColorEffectscomboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Effects</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ManualISOcheckBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual ISO</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="BrightspinBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ExposureTimespinBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ContrspinBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSlider" name="BrightSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="ExposureTimehorizontalSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Saturation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>277</x>
        <y>310</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Scene Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Exposure Metering Mode</string>
      </property>
     </widget>
     <widget class="QComboBox" name="AutoExpBiascomboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Red Balance</string>
      </property>
     </widget>
     <widget class="QSlider" name="BlueBalancehorizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Contrast</string>
      </property>
     </widget>
     <widget class="QSlider" name="ContrSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="RedBalanceSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="WhiteBalanceAPcomboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ExposureMeteringcomboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="BlueBalancespinBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7999</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="ManualExposurecheckBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual Exposure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Exposure Bias</string>
      </property>
     </widget>
     <widget class="QComboBox" name="IsoSensitivitycomboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="SaturationspinBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue Balance</string>
      </property>
     </widget>
     <widget class="QSlider" name="SaturationSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>White Balance Auto Preset</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Preprocesing</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
